llvm/test/Transforms/SLPVectorizer/RISCV/ctpop.ll

```diff
@@ -0,0 +1,107 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 2
+; RUN: opt < %s -passes=slp-vectorizer -S -mtriple=riscv64 -mattr=+v,+f | FileCheck %s
```

Comment:
Test more attribute variants?

Reply:
I don't know much about riscv, but maybe similar RUNs as found in the rvv/fixed-vectors-ctpop.ll codegen tests?

Reply:
I added a bunch of RUN lines, although the CHECK prefixes are unused: the codegen is identical. I hope this resolves the issue.